### ISSUE TITLE
NVD report shows a vulnerability

### ISSUE THREAD

**Issue by jeaye** [aprobus/jerks-whistling-tunes#4]
Hey there. Looks like `0.3.2` shows up as vulnerable using `lein nvd`, due to a public CVE. The output is:

```text
... elided ...
| jerks-whistling-tunes-0.3.2.jar | CVE-2016-7037 |
... elided ...
```

A quick link to that CVE is [here](https://nvd.nist.gov/vuln/detail/CVE-2016-7037). It applies to JWT, for sure, but I'm not certain if it's a false positive for the latest version of this lib. Got any more info?

**Comment by aprobus** [comment]
Hey @jeaye ! It looks like a false positive to me as the dependencies for this project all pass `lein nvd check` and I don't think this library has any overlap with a php application. I'll have to look more into this, thanks for bringing it up!

**Comment by jeaye** [comment]
Indeed, I ran `lein nvd check` for this repo as well and all was green. I'm not sure what the process of ruling out these false positives is, aside from explicitly ignoring them within lein-nvd; ideally everyone using jerks-whistling-tunes doesn't have to do that.
